docs/developers/celestia-app.md

```diff
@@ -0,0 +1,71 @@
+# Celestia App
+
+This tutorial will guide you through building Celestia App. This
+tutorial presumes you completed the steps in setting up your
+own environment [here](../developers/environment).
+
+## Install Celestia App
+
+The steps below will create a binary file named `celestia-appd`
+inside `$HOME/go/bin` folder which will be used later to run the node.
+
+```sh
+cd $HOME
+rm -rf celestia-app
+git clone https://github.com/celestiaorg/celestia-app.git
+cd celestia-app/
+APP_VERSION=$(curl -s \
+  https://api.github.com/repos/celestiaorg/celestia-app/releases/latest \
+  | jq -r ".tag_name")
+git checkout tags/$APP_VERSION -b $APP_VERSION
+make install
+```
+
+To check if the binary was successfully compiled you can run the binary
+using the `--help` flag:
+
+```sh
+celestia-appd --help
+```
+
+You should see a similar output (with helpful example commands):
+
+```text
+Stargate CosmosHub App
```

Comment:
I think this shows:
```suggestion
Start celestia app
```

@evan-forbes can confirm.

Reply:
I pushed up a commit to fix.